mem-01-b.xml

```diff
@@ -11750,7 +11750,7 @@ The 2009 {Star Trek:src} movie has a deleted scene with the line: {buqmeylIjDaq
       <column name="notes_pt">{Su'lop:n} alimentado a um animal e {Suqqa':v}. [AUTOTRANSLATED]</column>
       <column name="notes_fi">{Su'lop:n} syötetään eläimelle ja {Suqqa':v}. [AUTOTRANSLATED]</column>
       <column name="hidden_notes">The K-E side says [kud] instead of [qud].</column>
-      <column name="components"></column>
+      <column name="components">{burgh:n}, {quD:n}</column>
```

Comment:
This also requires `deriv` to be added `part_of_speech` above. To be consistent, **'un quD** should be marked `fic`.